--- pdfium/core/fpdfapi/font/cpdf_simplefont.cpp	2025-11-20 14:44:25
+++ core/fpdfapi/font/cpdf_simplefont.cpp	2025-12-03 16:39:54
@@ -94,7 +94,7 @@
   char_bbox_[charcode] = face->GetGlyphBBox();
   FXFT_FaceRec* face_rec = face->GetRec();
   if (use_font_width_) {
-    int TT_Width = NormalizeFontMetric(face_rec->glyph->metrics.horiAdvance,
+    int TT_Width = NormalizeFontMetric(FXFT_Get_Glyph_HoriAdvance(face_rec),
                                        face->GetUnitsPerEm());
     if (char_width_[charcode] == 0xffff) {
       char_width_[charcode] = TT_Width;
