--- pdfium/core/fxge/freetype/fx_freetype.h	2025-11-20 14:44:25
+++ core/fxge/freetype/fx_freetype.h	2025-12-03 16:39:54
@@ -57,6 +57,12 @@
   const pdfium::span<const FT_Var_Axis> axis_;
 };
 
+#define FXFT_Get_Glyph_HoriBearingX(face) (face)->glyph->metrics.horiBearingX
+#define FXFT_Get_Glyph_HoriBearingY(face) (face)->glyph->metrics.horiBearingY
+#define FXFT_Get_Glyph_Width(face) (face)->glyph->metrics.width
+#define FXFT_Get_Glyph_Height(face) (face)->glyph->metrics.height
+#define FXFT_Get_Glyph_HoriAdvance(face) (face)->glyph->metrics.horiAdvance
+
 int FXFT_unicode_from_adobe_name(const char* glyph_name);
 void FXFT_adobe_name_from_unicode(pdfium::span<char> name_buf, wchar_t unicode);
 
