--- pdfium/core/fxge/cfx_font.cpp	2025-11-20 14:44:25
+++ core/fxge/cfx_font.cpp	2025-12-03 16:39:54
@@ -72,6 +72,28 @@
                          right * 1000 / x_scale, bottom * 1000 / y_scale);
 }
 
+#ifdef PDF_ENABLE_XFA
+unsigned long FTStreamRead(FXFT_StreamRec* stream,
+                           unsigned long offset,
+                           unsigned char* buffer,
+                           unsigned long count) {
+  if (count == 0) {
+    return 0;
+  }
+
+  IFX_SeekableReadStream* pFile =
+      static_cast<IFX_SeekableReadStream*>(stream->descriptor.pointer);
+
+  // SAFETY: caller ensures `buffer` points to at least `count` bytes.
+  return pFile && pFile->ReadBlockAtOffset(
+                      UNSAFE_BUFFERS(pdfium::span(buffer, count)), offset)
+             ? count
+             : 0;
+}
+
+void FTStreamClose(FXFT_StreamRec* stream) {}
+#endif  // PDF_ENABLE_XFA
+
 bool ShouldAppendStyle(const ByteString& style) {
   return !style.IsEmpty() && style != "Regular";
 }
@@ -175,9 +197,29 @@
 bool CFX_Font::LoadFile(RetainPtr<IFX_SeekableReadStream> pFile,
                         int nFaceIndex) {
   object_tag_ = 0;
-  face_ = CFX_Face::OpenFromStream(
-      CFX_GEModule::Get()->GetFontMgr()->GetFTLibrary(), pFile, nFaceIndex);
-  return !!face_;
+
+  auto pStreamRec = std::make_unique<FXFT_StreamRec>();
+  pStreamRec->base = nullptr;
+  pStreamRec->size = static_cast<unsigned long>(pFile->GetSize());
+  pStreamRec->pos = 0;
+  pStreamRec->descriptor.pointer = static_cast<void*>(pFile.Get());
+  pStreamRec->close = FTStreamClose;
+  pStreamRec->read = FTStreamRead;
+
+  FT_Open_Args args;
+  args.flags = FT_OPEN_STREAM;
+  args.stream = pStreamRec.get();
+
+  face_ = CFX_Face::Open(CFX_GEModule::Get()->GetFontMgr()->GetFTLibrary(),
+                         &args, nFaceIndex);
+  if (!face_) {
+    return false;
+  }
+
+  owned_file_ = std::move(pFile);
+  owned_stream_rec_ = std::move(pStreamRec);
+  face_->SetPixelSize(0, 64);
+  return true;
 }
 
 #if !BUILDFLAG(IS_WIN)
@@ -313,12 +355,12 @@
     return std::nullopt;
   }
   int em = face_->GetUnitsPerEm();
-  return ScaledFXRectFromFTPos(face_->GetRec()->glyph->metrics.horiBearingX,
-                               face_->GetRec()->glyph->metrics.horiBearingY -
-                                   face_->GetRec()->glyph->metrics.height,
-                               face_->GetRec()->glyph->metrics.horiBearingX +
-                                   face_->GetRec()->glyph->metrics.width,
-                               face_->GetRec()->glyph->metrics.horiBearingY, em,
+  return ScaledFXRectFromFTPos(FXFT_Get_Glyph_HoriBearingX(face_->GetRec()),
+                               FXFT_Get_Glyph_HoriBearingY(face_->GetRec()) -
+                                   FXFT_Get_Glyph_Height(face_->GetRec()),
+                               FXFT_Get_Glyph_HoriBearingX(face_->GetRec()) +
+                                   FXFT_Get_Glyph_Width(face_->GetRec()),
+                               FXFT_Get_Glyph_HoriBearingY(face_->GetRec()), em,
                                em);
 }
 
