--- pdfium/core/fxge/cfx_fontmapper.cpp	2025-11-20 14:44:25
+++ core/fxge/cfx_fontmapper.cpp	2025-12-03 21:15:42
@@ -918,19 +918,19 @@
   size_t font_offset = ttc_size - data_size;
   size_t face_index =
       GetTTCIndex(font_desc->FontData().first(ttc_size), font_offset);
-  RetainPtr<CFX_Face> face(font_desc->GetFace(face_index));
-  if (face) {
-    return face;
+  RetainPtr<CFX_Face> pFace(font_desc->GetFace(face_index));
+  if (pFace) {
+    return pFace;
   }
 
-  face = font_mgr_->NewFixedFace(
+  pFace = font_mgr_->NewFixedFace(
       font_desc, font_desc->FontData().first(ttc_size), face_index);
-  if (!face) {
+  if (!pFace) {
     return nullptr;
   }
 
-  font_desc->SetFace(face_index, face.Get());
-  return face;
+  font_desc->SetFace(face_index, pFace.Get());
+  return pFace;
 }
 
 RetainPtr<CFX_Face> CFX_FontMapper::GetCachedFace(void* font_handle,
@@ -950,19 +950,19 @@
     font_desc = font_mgr_->AddCachedFontDesc(subst_name, weight, is_italic,
                                              std::move(font_data));
   }
-  RetainPtr<CFX_Face> face(font_desc->GetFace(0));
-  if (face) {
-    return face;
+  RetainPtr<CFX_Face> pFace(font_desc->GetFace(0));
+  if (pFace) {
+    return pFace;
   }
 
-  face = font_mgr_->NewFixedFace(font_desc,
-                                 font_desc->FontData().first(data_size), 0);
-  if (!face) {
+  pFace = font_mgr_->NewFixedFace(font_desc,
+                                  font_desc->FontData().first(data_size), 0);
+  if (!pFace) {
     return nullptr;
   }
 
-  font_desc->SetFace(0, face.Get());
-  return face;
+  font_desc->SetFace(0, pFace.Get());
+  return pFace;
 }
 
 // static
